arcaflow_plugin_uperf/uperf_schema.py

```diff
@@ -504,10 +516,20 @@ class UPerfServerParams:
         default=60,
         metadata={
             "name": "run_duration",
-            "description": "How long the server should run before terminating."
+            "description": "How long the UPerf server should run before terminating."
             " 0 for indefinite.",
         },
     )
+    comm_port: int = field(
+        default=20000,
+        metadata={
+            "name": "comm_port",
+            "description": "The master communication port to open for "
+                           "this UPerf server. This is used for initial "
+                           "communication with the server (slave).\n"
+                           "Must match the value specified in the client.",
```

Comment:
This text is quite good.  It seems like you should use (nearly) the same text at lines 502-505.

(FWIW, this port is actually used throughout the test, not just "initially", so I would drop the word "initial".)

Reply:
I slightly modified this, then copied it to the other instance and updated it to make more sense in the client side documentation.